### PR TITLE
JS test

### PR DESCRIPTION
**Pull request by dsekulovic**

### CODE REVIEW

File: desko.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desko page</title>
+    <script src="index.js"></script>
+</head>
+
+<body>
+    <h1>Desko</h1>
+</body>
+
+</html>
```
Comment (BraneOpacic21) [inline desko.html:15]: There should be a `/n` here, but not an issue now. Just, for future reference.

**Review by BraneOpacic21 — APPROVED** [review]
We will take a look at the given task, for now - it is approved to be merged (so its on the dev)!

File: desko.html (hunk 1 above)
Comment (dsekulovic) [reply desko.html:15]: Should I merge to develop?

Comment (BraneOpacic21) [reply desko.html:15]: Sure 👍

**Pull request merged**